src/main/java/io/cdap/e2e/pages/actions/CdfPluginPropertiesActions.java

```diff
@@ -460,6 +460,23 @@ public static void clickOnTab(String tabName) {
     Assert.assertTrue("Schema column list should match the preview column list",
       previewTableColumnList.equals(listOfFieldsInInputSchema));
   }
+  /**
+   * Get Plugin Property input webelement
+   *
+   * @param pluginProperty @data-cy attribute value of Plugin Property.
+   *                       If pluginProperty is present in {@link ConstantsUtil#DEFAULT_DATACY_ATTRIBUTES_FILE}
+   *                       then its data-cy is fetched from it
+   *                       else pluginProperty is used as it is.
+   * @return
+   */
+  public static WebElement getInputPluginPropertyElement(String pluginProperty) {
```

Comment:
move this method to `PluginPropertyUtils` class.
And reuse it in `replaceValueInInputProperty(String pluginProperty, String value)` method as well.

Reply:
Moved the method to **PluginPropertyUtils** class and re-used in **replaceValueInInputProperty**(String pluginProperty, String value) method (23e21cd29f01c71b0fecee3cf1a6b5b41fd74c03)